broker-daemon/orderbook/price-index.js

```diff
@@ -42,18 +45,22 @@ class PriceIndex extends SublevelIndex {
 
   /**
    * Placeholder for implementations of the price index
-   * @param {string} quantumPrice - Decimal of the quantumPrice
+   * @param {string} _quantumPrice - Decimal of the quantumPrice
+   * @returns {string}
    * @throws {Error} keyForPrice must be implemented by child class
    */
-  keyForPrice (quantumPrice) {
-    throw new Error('`keyForPrice` must be implemented by child classes.')
+  keyForPrice (_quantumPrice) {
+    if (_quantumPrice || !_quantumPrice) {
+      throw new Error('`keyForPrice` must be implemented by child classes.')
+    }
+    return ''
   }
 
   /**
    * Create a read stream of orders with prices at least as good as the given one
    * Note: If no quantumPrice is provided, creates a read stream of all records
-   * @param {string} quantumPrice - Decimal of the price
-   * @returns {ReadableStream}       ReadableStream from sublevel-index
+   * @param {?string} quantumPrice - Decimal of the price
+   * @returns {Sublevel}
```

Comment:
I think this returns a `ReadableStream`, not a `Sublevel`

Reply:
If it is `ReadableStream` then typescript says `Property 'on' does not exist on type 'ReadableStream<any>'`. It looks like it might be sublevel has an extended ReadableStream?

Reply:
It's a nodejs stream.Readable, not sure how we can represent that.